assets/blocks/course-list-block/index.js

```diff
@@ -71,13 +75,21 @@ const observeAndRemoveSettingsFromPanel = ( blockSettingsPanel ) => {
 	const observer = new MutationObserver( () => {
 		const selectedBlock = select( 'core/block-editor' ).getSelectedBlock();
 
+		if ( 'core/query' !== selectedBlock?.name ) {
+			return;
+		}
+
 		if (
-			'core/query' === selectedBlock?.name &&
 			selectedBlock?.attributes?.className?.includes(
 				'wp-block-sensei-lms-course-list'
 			)
 		) {
 			hideUnnecessarySettingsForCourseList();
+		} else if ( document.querySelector( '.border-block-support-panel' ) ) {
```

Comment:
I think we can remove this now too?

Reply:
Thanks! Removed [here](https://github.com/Automattic/sensei/pull/5576/commits/de3e34de836ba53e7a4cd0bf45bcd2f1a6c0e816)